old-ui/app/components/add-token/token-list/token-list.component.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import { checkExistingAddresses } from '../util'
+import TokenListPlaceholder from './token-list-placeholder'
+import h from 'react-hyperscript'
+import Tooltip from '../../tooltip.js'
+
+export default class InfoBox extends Component {
+  static contextTypes = {
+    t: PropTypes.func,
+  }
+
+  static propTypes = {
+    tokens: PropTypes.array,
+    results: PropTypes.array,
+    selectedTokens: PropTypes.object,
+    onToggleToken: PropTypes.func,
+  }
+
+  render () {
+    const { results = [], selectedTokens = {}, onToggleToken, tokens = [] } = this.props
+
+    return results.length === 0
+      ? <TokenListPlaceholder />
+      : (
+        <div className="token-list">
+          <div className="token-list__title">
+            { 'searchResults' /* this.context.t('searchResults')*/ }
+          </div>
+          <div className="token-list__tokens-container">
+            {
+              Array(6).fill(undefined)
+                .map((_, i) => {
+                  const { logo, symbol, name, address } = results[i] || {}
+                  const tokenAlreadyAdded = checkExistingAddresses(address, tokens)
+                  const title = `${name} (${symbol})`
+                  const isLongTitle = title.length > 28
+
+                  const tokenRow = (key) => h(`.${classnames('token-list__token', {
```

Comment:
I think we should use JSX here too to be consistent with the rest of the file

Reply:
Thank you, agree. react-hyperscript was changed to JSX.